cases/0010/_model.xml

```diff
@@ -37,6 +39,11 @@
     </listOfParameters>
 
     <listOfInitialAssignments>
+      <initialAssignment symbol="A">
```

Comment:
I think I left one without initial assignment on purpose, to check that both are working. Need to think if that really covers a relevant case. Not sure if it is well defined what should happen if there is the species in the condition table. I guess the assignment should have priority.

Reply:
@FFroehlich : Was there a specific reason to add this initialAssignment?

Reply:
Uhm, honestly I have no idea.

Reply:
Okay, I'll revert all changes to the SBML models due to unclear motivation.

Reply:
Ah... Problem found. Some tests were supposed to use `conversion_modified.xml` instead of `conversion.xml`, which you seem to have kicked out :grimacing: